.gitignore

```diff
@@ -39,3 +39,29 @@
 /.sonar/
 /.scannerwork/
 /sonar-reports/
+
+# Created by https://www.toptal.com/developers/gitignore/api/vim
```

Comment:
Can we remove the 3 toptal lines?

Reply:
Not really sure why any of this is here anyhow. I use vim and do not need this

Reply:
We can of course. This is there mostly because I did some dirty things and add a swap file, and I have the reflex of `gi vim >> .gitignore` when I see a swap file in the workdir. No prob to remove that if this is not ok with you.